core/vibes/soul/sections/products-list-section/sorting.tsx

```diff
@@ -41,11 +44,13 @@ function SortingInner({
   defaultValue,
   options: streamableOptions,
   label: streamableLabel,
+  placeholder: streamablePlaceholder,
```

Comment:
Looks like we're still not passing this value from the places that reference this component